INSTALL.md

```diff
@@ -140,19 +140,51 @@ pecl install oci8-2.0.10
 ```
 This will download and compile the module, and place it in your php extension dir.
 
-Add the ```extension=oci8.so``` line to your php.ini or create a configuration file: 
+Add the ```extension=oci8.so``` line to your php.ini or create a configuration file:
 
-```bash 
+```bash
 echo 'extension=oci8.so' > /etc/php.d/oci8.ini
 ```
 
 Confirm it is working with ```php -i | grep -i oci8```
 
+#### MariaDB/mysql
+The following instructions are to set up a local MYSQL database for GOCDB. They have not been tested in a production enviroment and are currently intended for test instances.
+
+First you will need to install MariaDB. Then start the 'mysqld' service.
```

Comment:
you need to install MariaDB server as well

Reply:
And under MariaDB the service you need to start is `mariadb` not `mysqld` - at least on SL7

Reply:
As in MariaDB server, and then client?

Reply:
yes, although the "client" package is just called `mariadb`.

So `mariadb-server` and `mariadb` are the needed packages on SL7

Reply:
Ok, open another PR, building on this one?

Reply:
```suggestion
First you will need to install the MariaDB server and client. Then start the 'mysqld' service.
```